master/buildbot/db/buildrequests.py

```diff
@@ -224,6 +227,43 @@ def thd(conn):
 
         return self.db.pool.do(thd)
 
+    def getBuilrequestsOwnedBy(self, owner):
+        if not (isinstance(owner, str) or isinstance(owner, unicode)):
```

Comment:
btw we should probably limit the result to a number of builds and sort it to the most recent in the backend

Reply:
yeah thats the idea